saleor/core/templatetags/shop.py

```diff
@@ -27,3 +28,38 @@ def get_sort_by_url(context, field, descending=False):
     else:
         request_get['sort_by'] = field
     return '%s?%s' % (request.path, urlencode(request_get))
+
+
+@register.assignment_tag(takes_context=True)
+def get_sort_by_toggle(context, field):
+    '''
+    This templatetag returns data needed for sorting querysets by links.
+    '''
+    request = context['request']
+    request_get = request.GET.copy()
+    sort_by = request_get.get('sort_by')
+    arrow_src = ''
+    active = ''
+    if sort_by:
+        # toggle existing sort_by
+        if field == sort_by:  # descending sort
+            new_sort_by = u'-%s' % field
+            arrow_src = static('/images/arrow_up_icon.svg')
+            active = 'active'
+        else:  # ascending sort
+            new_sort_by = field
+            if field == sort_by[1:]:
+                arrow_src = static('/images/arrow_down_icon.svg')
+                active = 'active'
+            else:
```

Comment:
When this case happens? Isn't setting of these values already covered in lines 41 and 42?

Reply:
Yes, some old code i guess. Deleted